functions/Restore-DbaDatabase.ps1

```diff
@@ -583,7 +583,12 @@ function Restore-DbaDatabase {
                     $Completed = 'successfully'
                 }
                 catch {
-                    Write-Exception $_
+                    if ($_.FullyQualifiedErrorId -like 'Destination File*' ) {
+                        Stop-Function -Message "Restore of $databasename failed, $($_.FullyQualifiedErrorId) " -InnerErrorRecord $_.Exception.InnerException -silent:$silent    
+                    }
+                    else {
+                        Stop-Function -Message "Restore of $databasename failed" -InnerErrorRecord $_.Exception.InnerException -silent:$silent
```

Comment:
Should instead be:
`Stop-Function -Message "Restore of $databasename failed" -ErrorRecord $_ -Silent $silent`